watcher/tool/watcher/rollups.go

```diff
@@ -30,8 +30,8 @@ import (
 	"k8s.io/apimachinery/pkg/watch"
 )
 
-func runRollupsWatcher(ctx context.Context, kubernetesClient *kubernetes.Client, retryC chan<- func() error) error {
-	influxDBClient, err := influxdb.NewClient()
+func runRollupsWatcher(ctx context.Context, kubernetesClient *kubernetes.Client, config influxdb.Config, retryC chan<- func() error) error {
```

Comment:
Can the rest of the arguments be moved to the config parameter? 4 arguments is two too many, imho.